themes/bootstrap3/templates/header.phtml

```diff
@@ -13,7 +13,8 @@
 <? endif; ?>
 <? if (!isset($this->layout()->renderingError)): ?>
   <div class="collapse navbar-collapse" id="header-collapse">
-    <ul role="navigation" class="nav navbar-nav navbar-right flip">
+    <nav>
+    <ul class="nav navbar-nav navbar-right flip">
```

Comment:
This `<ul>` and its contents should probably be indented two spaces to keep the tag nesting consistent.